authors/settings.py

```diff
@@ -78,6 +78,19 @@
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
+if 'TRAVIS' in os.environ:
+    DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```

Comment:
Can you please pick these from the `environment`

Reply:
I have worked on the issue